org.kde.kaffeine.json

```diff
@@ -249,18 +259,8 @@
       "sources": [
         {
           "type": "archive",
-          "url": "https://download.kde.org/stable/kaffeine/kaffeine-2.0.18.tar.xz",
-          "sha256": "d9799a5b7b94a55963f94938b36b8fcb1be3e753b2be110989934ab15386b681",
-          "x-checker-data": {
-            "type": "anitya",
-            "project-id": 5557,
-            "stable-only": true,
-            "url-template": "https://download.kde.org/stable/kaffeine/kaffeine-$version.tar.xz"
-          }
-        },
-        {
-          "type": "patch",
-          "path": "appdata.patch"
+          "url": "https://invent.kde.org/multimedia/kaffeine/-/archive/v2.0.19/kaffeine-v2.0.19.tar.gz",
```

Comment:
You can not do this. KDE system administrators will be very very very unhappy if you do this.

Reply:
can we use git?

Reply:
Yes, it is to prevent overload, and also because it is the right thing to do.

You are right runtimes fetch frameworks for invent, and to be honest it is wrong, i'm just to tired to fight that, but let's try to do the right thing here, if we can't get the kaffeine folks to do it right, i guess we could live with with using git.

Reply:
ok, i emailed the person that did the tag and they have not answered me back, let's go with this i guess and hope the sysadmins don't hate us a lot :D